src/notebook/Notebook.jl

```diff
@@ -54,10 +54,26 @@ Base.@kwdef mutable struct Notebook
     wants_to_interrupt::Bool=false
     last_save_time::typeof(time())=time()
     last_hot_reload_time::typeof(time())=zero(time())
+    last_serialized_version::String = ""
+    write_out_fs::Bool=true
```

Comment:
We already have https://github.com/fonsp/Pluto.jl/blob/c1ba792fc154c306129b58eb723621544b91f59c/src/Configuration.jl#L58 for this

Reply:
this is notebook global and it prevents the save function from running at all. this flag is per-notebook and it only prevents the final write-out to the file. Thus, we can run save event listeners and have the latest text representation of the file